internal/controller/nimcache_controller.go

```diff
@@ -910,6 +911,20 @@ func getManifestConfigName(nimCache *appsv1alpha1.NIMCache) string {
 	return fmt.Sprintf("%s-manifest", nimCache.GetName())
 }
 
+func getCommand() []string {
+	// Default model manifest file path
+	defaultManifestFile := "/etc/nim/config/model_manifest.yaml"
+	updatedManifestFile := "/opt/nim/etc/default/model_manifest.yaml"
+
+	// Check for the manifest file at the updated path with latest NIMs
+	if _, err := os.Stat(updatedManifestFile); err == nil {
```

Comment:
Does the controller have access to the nim file path here?

Reply:
Thanks Shi, i have updated this.